src/TAlias.cpp

```diff
@@ -299,10 +305,30 @@ void TAlias::setRegexCode( QString code )
     if( re == NULL )
     {
         mOK_init = false;
+        if( mudlet::debugMode ) {
+            TDebug( QColor(Qt::white), QColor(Qt::red) ) << "REGEX ERROR: failed to compile, reason:\n"
+                                                         << error
+                                                         << "\n"
+                                                         >> 0;
+            TDebug( QColor(Qt::red), QColor(Qt::gray) ) << "in: \""
+                                                        << mRegexCode
+                                                        << "\"\n"
+                                                        >> 0;
+        }
+        setError( tr( "Error: in \"Pattern:\", faulty regular expression, reason: \"%1\"." )
```

Comment:
Great error message! I really like how it points straight to the problem field in trigger patterns and aliases. Perhaps in the future we could highlight the field as well like web validation does.

Would you mind setting the text colour to blue, since consistency is [super important](https://www.interaction-design.org/literature/article/user-interface-design-guidelines-10-rules-of-thumb) in user interfaces:

![lusternia_023](https://cloud.githubusercontent.com/assets/110988/23642917/a904ccc8-02fe-11e7-90c3-9bb5f13dc71b.png)

![lusternia_022](https://cloud.githubusercontent.com/assets/110988/23642945/c718d006-02fe-11e7-97b9-6eddfa7bb40a.png)

I understand this fixes it to work on dark themes, but there are other issues in dark themes as well so this doesn't help much - let's put all the dark theme fixes into one release so we can preserve consistency (and at least only break it between releases).